docs/content/preview/yugabyte-voyager/reference/cutover-archive/archive-changes.md

```diff
@@ -30,6 +30,6 @@ The valid *arguments* for archive changes are described in the following table:
 | --fs-utilization-threshold <percentage> | Disk utilization threshold in percentage. (default: 70) |
 | -h, --help | Command line help for archive changes. |
 | --move-to <path> | Destination path to move exported data to. |
-| --send-diagnostics | Send diagnostics information to Yugabyte. (default: true) |
+| --send-diagnostics | Send [diagnostics](../../../diagnostics-report/) information to Yugabyte. (default: true) |
```

Comment:
cutover commands don't have this flag?

Reply:
cutover commands don't have it as of version 1.5. @priyanshi-yb any plans of including it later ?

Reply:
No, not yet. will add this in the future if required.